src/test/test-capability.c

```diff
@@ -181,6 +181,12 @@ static void test_update_inherited_set(void) {
         caps = cap_get_proc();
         assert_se(caps);
         assert_se(!cap_get_flag(caps, CAP_CHOWN, CAP_INHERITABLE, &fv));
+
+        if (fv == CAP_SET) {
+                log_info("%s: Inheritable flag is already set, skipping test", __FUNCTION__);
+                return;
+        }
```

Comment:
let's just drop the precondition check alltogether, if it's not universally true. i.e. drop the cap_get_flag() assert above, and the == CAP_CLEAR assert below.

if the preconditions don't hold, let's not do them. We can still run the rest of the test and verify that the effect is systematics regardless what the preconditions are...

Reply:
makes sense, fixed